examples/rapids-blog/.saturn/saturn.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "example-rapids-blog",
```

Comment:
Can the folder name be `rapids-first-blog` and then the resource name be `example-rapids-first-blog`? I want to make it clear since we have multiple blog posts on RAPIDS